src/item.h

```diff
@@ -1777,5 +1777,10 @@ enum hint_rating {
  */
 item &null_item_reference();
 
+/**
+ * Returns freezing point temperature of item, mainly for non-water freezing points
+ */
+int check_freezing_temperature( const item &it );
```

Comment:
Still, I'd turn this function into a member function. Like `int item::get_freezing_temperature() const`.

Reply:
I dont have enough experience in coding, so don't mind me asking: is it any better as member of item then a free functin? Such nuances are not entirely clear to me. And sure, I can do that, but ETA is 10 days.

Reply:
It's just more OOP-ish way which has some benefits. Here's some of them:
- This way you clearly state that it's a part of the `item`s interface (the function can then be made `virtual` if needed);
- Access to the cuts of the `class` (`private` and `protected` stuff);
- No need to reference the argument all the time (like `it.foo`, `it.bar`);

Reply:
There's no performance-wise difference: the argument (a pointer to the class) is implicitly passed nevertheless.